src/app/weather-data/weather-data.component.scss

```diff
@@ -1,7 +1,22 @@
 h1 {
   background-color: #3b3b3b;
-  width: 100vw;
+  width: 98.02%;
```

Comment:
might want to take a look at `box-sizing` as a css property, you can prevent padding affecting the width of an element